specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/costmanagement.json

```diff
@@ -0,0 +1,2577 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.CostManagement/{externalCloudProviderType}/{externalCloudProviderId}/forecast": {
+      "post": {
+        "tags": [
+          "Forecast"
+        ],
+        "operationId": "Forecast_ExternalCloudProviderUsage",
+        "description": "Lists the forecast charges for external cloud provider type defined.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "ExternalBillingAccountForecast": {
+            "$ref": "./examples/ExternalBillingAccountForecast.json"
+          },
+          "ExternalSubscriptionForecast": {
+            "$ref": "./examples/ExternalSubscriptionForecast.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "$filter",
+            "description": "May be used to filter forecasts by properties/usageDate (Utc time), properties/chargeType or properties/grain. The filter supports 'eq', 'lt', 'gt', 'le', 'ge', and 'and'. It does not currently support 'ne', 'or', or 'not'.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/externalCloudProviderTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/externalCloudProviderIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ForecastDefinition"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate Forecast Config operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/QueryResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/dimensions": {
+      "get": {
+        "tags": [
+          "Dimensions"
+        ],
+        "x-ms-odata": "#/definitions/Dimension",
+        "operationId": "Dimensions_List",
+        "description": "Lists the dimensions by the defined scope.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "SubscriptionDimensionsList-Legacy": {
+            "$ref": "./examples/SubscriptionDimensionsList.json"
```

Comment:
This example has an expand parameter - lets remove it.

Reply:
listing all examples that need to be removed because of having expand:
 
SubscriptionDimensionsList
ResourceGroupDimensionsList
ManagementGroupDimensionsListWithFilter
DepartmentDimensionsListWithFilter
EnrollmentAccountDimensionsListWithFilter
BillingAccountDimensionsListWithFilter
MCABillingAccountDimensionsListExpandAndTop
MCABillingAccountDimensionsListWithFilter
MCABillingProfileDimensionsListWithFilter
MCAInvoiceSectionDimensionsListWithFilter
MCACustomerDimensionsListWithFilter

Reply:
@handeeeadiguzel  removed all the examples and closing on this one